src/medCore/gui/commonWidgets/medSlider.cpp

```diff
@@ -0,0 +1,55 @@
+/*=========================================================================
+
+ medInria
+
+ Copyright (c) INRIA 2013 - 2014. All rights reserved.
+ See LICENSE.txt for details.
+ 
+  This software is distributed WITHOUT ANY WARRANTY; without even
+  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+  PURPOSE.
+
+=========================================================================*/
+
+#include "medSlider.h"
+
+
+medSlider::medSlider() : QSlider()
+{
+}
+
+medSlider::~medSlider()
```

Comment:
Since both the constructor and the destructor are empty, there was no point in adding them.